src/libPMacc/examples/gameOfLife2D/CMakeLists.txt

```diff
@@ -131,84 +57,22 @@ endif(VAMPIR_ENABLE)
 option(GOL_RELEASE "Build release version, disables all runtime asserts" OFF)
 if(GOL_RELEASE)
     message(STATUS "Release version")
-
     set(CMAKE_BUILD_TYPE Release)
     add_definitions(-DNDEBUG)
-    set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}" "-Xcompiler=-pthread")
 else(GOL_RELEASE)
     message(STATUS "Debug version")
-
     set(CMAKE_BUILD_TYPE Debug)
-    set(CUDA_NVCC_FLAGS "${CUDA_NVCC_FLAGS}" "-g" "-Xcompiler=-g,-pthread")
 endif(GOL_RELEASE)
 
 
-################################################################################
-# Find MPI
-################################################################################
-
-find_package(MPI REQUIRED)
-include_directories(SYSTEM ${MPI_C_INCLUDE_PATH})
-set(LIBS ${LIBS} ${MPI_C_LIBRARIES})
-
-# bullxmpi fails if it can not find its c++ counter part
-if(MPI_CXX_FOUND)
-    set(LIBS ${LIBS} ${MPI_CXX_LIBRARIES})
-endif(MPI_CXX_FOUND)
-
-
-################################################################################
-# Find PThreads
-################################################################################
-
-find_package(Threads REQUIRED)
-set(LIBS ${LIBS} ${CMAKE_THREAD_LIBS_INIT})
-
-
 ###############################################################################
 # Find Boost
 ###############################################################################
 
-find_package(Boost 1.49.0 REQUIRED COMPONENTS program_options regex system)
+find_package(Boost 1.49.0 REQUIRED COMPONENTS program_options )
 include_directories(SYSTEM ${Boost_INCLUDE_DIRS})
 set(LIBS ${LIBS} ${Boost_LIBRARIES})
 
-# work-arounds
```

Comment:
requireing that PMacc will take care of all boost work-arounds is a bit odd, because we might need more but ok-ish... thise one is outdated with 1.57+ anyway